### PR TITLE
Sort products

### PR DESCRIPTION
**Pull request by VedanthB**
- added sort products functionality

### CODE REVIEW

File: src/components/shop-page/FiltersSideBar.js [hunk 1]
```diff
@@ -1,6 +1,26 @@
...
+};
 
 function FiltersSidebar() {
+  const [state, dispatch] = useReducer(reducerFunc, {
+    jacketsChecked: false,
+    capsChecked: false,
+  });
+
+  console.log(state.jacketsChecked);
+
```
Comment (prathmeshjagtap) [inline src/components/shop-page/FiltersSideBar.js:23]: remove console.log

File: src/context/reducers/filterReducer.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+//   caps: false,
+//   skateboards: false,
+// },
+// brands: {
+//   vans: false,
+//   "urban-monkey": false,
+//   thrasher: false,
+//   "santa-cruz": false,
+// },
+// ratingsGreaterThan: 0,
```
Comment (prathmeshjagtap) [inline src/context/reducers/filterReducer.js:16]: remove commented code

File: src/routes/Shop.js [hunk 3]
```diff
@@ -1,9 +1,21 @@
...
 function Shop() {
+  const { state } = useProducts();
+
+  const { products } = state;
+
+  const { sortedProductsList } = useFilters();
+
+  console.log(sortedProductsList);
+
   return (
```
Comment (prathmeshjagtap) [inline src/routes/Shop.js:19]: remove console.log.

File: src/utils/getProducts.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
+import axios from "axios";
+
+import { productActions } from "../context/constants/productConstants";
+import { useProducts } from "../context/providers/ProductProvider";
+import { getProductsService } from "../services/productServices";
+
+export const getProducts = async (productsDispatch) => {
+  try {
+    console.log(productsDispatch);
+
```
Comment (prathmeshjagtap) [inline src/utils/getProducts.js:10]: remove conosole.log

File: src/utils/getProducts.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    const {
+      data: { products },
+      status,
+    } = await getProductsService();
+
+    console.log(status);
+
```
Comment (prathmeshjagtap) [inline src/utils/getProducts.js:17]: same here

**Review by prathmeshjagtap — APPROVED** [review]
Rest everything is good and well structured just remove commented code and console.logs before merging

File: src/context/constants/Test.js [hunk 6]
```diff
@@ -0,0 +1,193 @@
+import React, { useReducer } from "react";
+import "./styles.css";
+
+import faker from "faker";
+
+faker.seed(123);
+
+const data = [...Array(50)].map((item) => ({
```
Comment (SukanyaSen530) [inline src/context/constants/Test.js:8]: I guess you already have data, do you need this?

**Pull request merged**